library/common/http/dispatcher.h

```diff
@@ -168,19 +172,24 @@ class Dispatcher : public Logger::Loggable<Logger::Id::http> {
     const Network::Address::InstanceConstSharedPtr& connectionLocalAddress() override {
       return parent_.address_;
     }
+    absl::string_view responseDetails() override { return details_; }
     // TODO: https://github.com/lyft/envoy-mobile/issues/825
     void readDisable(bool /*disable*/) override {}
     uint32_t bufferLimit() override { return 65000; }
     // Not applicable
     void setFlushTimeout(std::chrono::milliseconds) override {}
 
+    void setDetails(absl::string_view details) { details_ = details; }
```

Comment:
`setResponseDetails`? I'm also curious as to what the benefit is of having an explicit setter when `details_` has the same access level below and can be modified directly

Reply:
yep, good eye. I actually want to turn all the members to private to prevent a conn manager <-> async stream situation here. So wanted to prepare for that.